### ISSUE TITLE
Android Error - Jitsi not found

### ISSUE THREAD

**Issue by jofftiquez** [calvinckho/capacitor-jitsi-meet#48]
When building in android I am getting this error:

> error: package com.capacitor.jitsi.plugin does not exist

I followed [this guide](https://github.com/calvinckho/capacitor-jitsi-meet/blob/master/android/README.md).

Here's my activity file:

```java
package org.mycure.mobile.pxp;

import android.os.Bundle;

import com.getcapacitor.BridgeActivity;
import com.getcapacitor.Plugin;
import com.capacitor.jitsi.plugin.Jitsi;

import java.util.ArrayList;

public class MainActivity extends BridgeActivity {
  @Override
  public void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);

    // Initializes the Bridge
    this.init(savedInstanceState, new ArrayList<Class<? extends Plugin>>() {{
      // Additional plugins you've installed go here
      // Ex: add(TotallyAwesomePlugin.class);
      add(Jitsi.class);
    }});
  }
}
```

**Dependencies**

```json
// main package.json
"@capacitor/core": "^2.4.7",
"capacitor-jitsi-meet": "^1.5.10",

// android package.json
"@capacitor/android": "2.4.7",
"@capacitor/core": "^2.4.7",
```

**Relevant screenshots**

![image](https://user-images.githubusercontent.com/8638243/115335265-68979980-a1cf-11eb-8b8c-c15b4a570ebf.png)

**Comment by calvinckho** [comment]
can you double check that your capacitor-jitsi-meet plugin is installed correctly?

In your android studio, do you see capacitor-jitsi-meet listed under your android project?

![Screen Shot 2021-04-20 at 1 47 57 PM](https://user-images.githubusercontent.com/23274587/115462427-7ce39280-a1df-11eb-9241-a4c1a140c273.png)

**Comment by jofftiquez** [comment]
@calvinckho thanks for the response. It seems that it's not there. I'm not an android developer, do I have to install the package manually?

Things I've already done:

1. `npx cap sync`
2. `npx cap copy` <- I'm not sure if this would help but I did it anyway :D

**Comment by calvinckho** [comment]
npm install capacitor-jitsi-meet should install it to your project. Then npx cap sync should add it to your android project.

**Comment by jofftiquez** [comment]
I'm using quasar, what I did is I manually re-installed `capacitor-jitsi-meet` in `src-capacitor` folder then did `npx cap sync`. The package is still not showing in android studio but the error went away. Altho there's a new error: 

> Could not find org.jitsi.react:jitsi-meet-sdk:3.4.0.
Required by:
    project :capacitor-jitsi-meet

**Comment by calvinckho** [comment]
I am not familiar with quasar. My app is in Ionic. You may want to read docs on how others install capacitor plugins in a quasar app. Here is the capacitor doc for your reference: https://capacitorjs.com/docs/v3/getting-started

**Comment by calvinckho** [comment]
If you want to see how I did it with my app, my full repo is at https://github.com/calvinckho/restvo-app and you are welcome to clone and build my app on your Android Studio and see if it sheds any lights.

**Comment by jofftiquez** [comment]
Thanks for the references @calvinckho I'll get back to you. Thanks.

**Comment by jofftiquez** [comment]
@calvinckho It's working now. Here's what I did wrong and how I corrected them:

1. I did not install the `capacitor-jitsi-meet` in the `src-capacitor` apparently, in the quasar the dependencies that you want to use in android/ios should be manually installed via yarn or nm in `src-capacitor` folder.

2. I misplaced the position of maven URL. Instead of putting it in `aallprojects > repositories` I put it in `buildScript > repositories`. 

Thank you for your time, your insights sure helped me. Thank you.
